sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -691,6 +739,58 @@ def _to_outgoing_message(self):
             to=self.to,
         )
 
+    def __repr__(self):
+        # type: () -> str
+        # pylint: disable=bare-except
+        message_repr = "body={}".format(
+            str(self.body)
+        )
+        try:
+            message_repr += ", application_properties={}".format(self.application_properties)
+        except:
```

Comment:
same except comment

Reply:
I really don't think we should align with it - given repr is 1024 chars long only - i don't think gives out any information important enough